ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -118,9 +124,9 @@ const ConfirmAddSuggestedToken = () => {
 
   const handleAddTokensClick = useCallback(async () => {
     await Promise.all(
```

Comment:
I've been seeing how to sort it, and I can't see an easy way out.

Since `acceptWatchAsset` doesn't throw when it fails to add the asset and it doesn't return anything, I cannot wrap it on a try/catch statement to reject the pending approval instead.